### ISSUE TITLE
Problems on Sublime Text build 3085 or newer

### ISSUE THREAD

**Issue by davidrios** [davidrios/jade-tmbundle#47]
https://www.sublimetext.com/forum/viewtopic.php?p=68210
The regex engine of Sublime Text was changed as of version 3 build 3085, and since this highlighter rely on some obscure features of the old engine, it might not work on the new one. I don't think it is possible to support both engines with the same bundle, I was pretty much forced to use those obscure features to implement this highlighter correctly. We will have to wait and see if they add some compatibility option to use the old engine, I'll ask for that on their forum.

tl;dr
Consider this bundle officially incompatible with dev builds of Sublime Text for the time being. If you have problems, stick with the latest stable release (8083).

**Comment by partap** [comment]
I just checked the website and it looks like they have rolled back to build 3083 as the latest version.  
Highlighting works correctly with this version.

**Comment by jayliu50** [comment]
I'm in build 3092 on Linux and Jade files don't seem to be highlighted at all. And I cannot choose Jade from the list of syntaxes. Does anyone happen to know why?

**Comment by davidrios** [comment]
This issue doesn't seem to exist anymore.
